testng.integration/src/main/java/ru/tinkoff/qa/neptune/testng/integration/DefaultTestRunningListener.java

```diff
@@ -67,17 +68,55 @@ private void refreshIfNecessary(Object instance, Method method) {
     public void beforeInvocation(IInvokedMethod method, ITestResult testResult) {
         ofNullable(testResult.getTestContext()).map(ITestContext::getSuite)
                 .ifPresent(GroupingObjects::addGroupingObjectForCurrentThread);
+        Method reflectionMethod = method.getTestMethod().getConstructorOrMethod().getMethod();
         ofNullable(testResult.getInstance()).ifPresent(o ->
-                refreshIfNecessary(o, method.getTestMethod().getConstructorOrMethod().getMethod()));
+                refreshIfNecessary(o, reflectionMethod));
+
+        ofNullable(reflectionMethod.getAnnotation(Test.class)).ifPresent(test -> {
+            String name = !isBlank(test.description()) ? test.description() : reflectionMethod.getName();
+            Object[] params = testResult.getParameters();
+
+            System.out.println();
+            System.out.println();
+            System.out.println(format("TEST '%s' HAS STARTED WITH PARAMETERS: %s", name, Arrays.toString(params)));
+            System.out.println();
+            System.out.println();
+        });
     }
 
     @Override
     public void afterInvocation(IInvokedMethod method, ITestResult testResult) {
-        if (!isIgnored(method.getTestMethod().getConstructorOrMethod()
-                .getMethod())) {
+        Method reflectionMethod = method.getTestMethod().getConstructorOrMethod().getMethod();
+        if (!isIgnored(reflectionMethod)) {
             Object instance = testResult.getInstance();
             knownTests.add(instance);
         }
+
+        ofNullable(reflectionMethod.getAnnotation(Test.class)).ifPresent(test -> {
+            String name = !isBlank(test.description()) ? test.description() : reflectionMethod.getName();
+            Object[] params = testResult.getParameters();
+
+            System.out.println();
+            System.out.println();
+            System.out.println(format("TEST '%s' HAS FINISHED WITH PARAMETERS: %s", name, Arrays.toString(params)));
+            int status = method.getTestResult().getStatus();
+            switch (status) {
+                case FAILURE:
+                    System.err.println("STATUS: FAILED. Exception:");
+                    testResult.getThrowable().printStackTrace();
+                    break;
+
+                case SKIP:
+                    System.out.println("STATUS: SKIPPED");
+                    break;
+
+                default:
+                    System.out.println("STATUS: SUCCEED. CONGRATULATIONS!!!!!!!");
```

Comment:
There's no call for using a few symbols like "!".
It will suffice making just single "!".